README.md

```diff
@@ -39,35 +51,131 @@ class ListMovingVehicle(vqpy.QueryBase):
     @staticmethod
     def setting() -> vqpy.VObjConstraint:
         filter_cons = {'__class__': lambda x: x == Vehicle,
-                       'bbox_velocity': lambda x: x >= 0.1}
+                       'velocity': lambda x: x >= 0.1}
         select_cons = {'track_id': None,
                        'license_plate': None}
         return vqpy.VObjConstraint(filter_cons=filter_cons,
                                    select_cons=select_cons)
 ```
 
+We also have a utility function named `vqpy.utils.continuing`, which checks whether a condition function continues to be true for a certain duration. The following code demonstrate a query that selects all the `Person` objects whose `bottom_center` property lies in the specified region. Then the `track_id` field, `coordinate` field and `in_roi_periods` field. Note that `in_roi_periods` is generated by the `vqpy.utils.continuing` instance, meaning that the total time period the condition is satisfied.
+
+```python
+class People_loitering_query(vqpy.QueryBase):
+
+    @staticmethod
+    def setting() -> vqpy.VObjConstraint:
+        REGIONS = [[
+            (550, 550), (1162, 400), (1720, 720), (1430, 1072), (600, 1073)
+        ]]
+
+        filter_cons = {
+            "__class__": lambda x: x == Person,
+            "bottom_center": vqpy.utils.continuing(
+                condition=vqpy.utils.within_regions(REGIONS),
+                duration=10, name="in_roi"
+            ),
+        }
+        select_cons = {
+            "track_id": None,
+            "coordinate": lambda x: str(x),  # convert to string for JSON serialization
+            "in_roi_periods": None,
+        }
+        return vqpy.VObjConstraint(filter_cons, select_cons)
+```
+
 #### Launch the task
 
-Last, we can call `vqpy.launch` to start query video frames.
+After declaring `VObj` and `Query`, users should call the `vqpy.launch` function to deploy the video query, below is an example of this interface. Users should specify the detector class they expect, and map the object detection result to the defined VObj types.
```

Comment:
what is detector class (`cls_type` or `detector_name`)? and the object detection result is? 
We should rephrase that to be more precise.